include/faabric/executor/FaabricMain.h

```diff
@@ -1,23 +1,31 @@
 #pragma once
 
-#include <faabric/executor/FaabricPool.h>
+#include <faabric/scheduler/ExecutorFactory.h>
```

Comment:
This now has the stuff that used to be in `FaabricPool` as well (i.e. managing the different servers)